Orange/widgets/data/utils/preprocess.py

```diff
@@ -412,7 +413,7 @@ def createWidgetFor(self, index):
         Create a QWidget instance for the index (:class:`QModelIndex`)
         """
         definition = index.data(DescriptionRole)
-        widget = definition.viewclass()
+        widget = definition.viewclass(self._parent)
```

Comment:
I checked how does the change in Controller influence my preprocessors in orange-spectroscopy. It is OK for most of them, but some assume that parent is a keyword argument and crash with your changes.

I though about changing mine, but then I saw that even QWidget uses parent as a keyword argument (see http://pyqt.sourceforge.net/Docs/PyQt4/qwidget.html). Therefore I propose changing this line to `widget = definition.viewclass(parent=self._parent)`

http://pyqt.sourceforge.net/Docs/PyQt4/qwidget.html

Reply:
Done.